infra/scripts/testnet.toml

```diff
@@ -107,7 +107,7 @@ export CMT_PERSISTENT_PEERS=`cat $BASE_DIR/peers`
 echo Persistent peers: $CMT_PERSISTENT_PEERS
 
 for i in $(seq 0 3); do
-    sed -i'bak' "s/persistent_peers = \\"\\"/persistent_peers = \\"$CMT_PERSISTENT_PEERS\\"/" $BASE_DIR/node${i}/cometbft/config/config.toml
+    sed -i'bak' "s|persistent_peers = \\"\\"|persistent_peers = \\"$CMT_PERSISTENT_PEERS\\"|" $BASE_DIR/node${i}/cometbft/config/config.toml
```

Comment:
Isn't `-e` required here? 
Why do you keep `bak` here but not [here](https://github.com/consensus-shipyard/fendermint/pull/373/files#diff-c7acb1a1a82d3b2fa4b0b12e343395aee71560f17d0a888a933fc3f2ecea4c3cR59)?

Reply:
Will fix